### DISCUSSION TITLE
Can not login in budibase builder.

### DISCUSSION THREAD

**Discussion by tooktok** [Budibase/budibase#15164] [Help]
Hi~
My company operates budibase as a "B to B web application" on 3 different environments(different servers, domains, budibase instances).
every budibase instances run with docker-compose(self hosted,  Premium license, using custom branding).
the budibase version is 2.21.9
a few days ago i found it became impossible to login into budibase builder.
in all 3 environments, budibase builder shows same error message when i try to login like following images

![image](https://github.com/user-attachments/assets/6f0b596c-3009-4e4b-9c02-eb29c541a4fe)

and here's docker logs (seems to be an hint)
```
{"level":"ERROR","timestamp":"2024-12-10T09:49:15.626Z","service":"@budibase/worker","err":{"type":"Error","message":"Error creating account user activity: Not Found","stack":"Error: Error creating account user activity: Not Found\n    at createActivity (/app/dist/index.js:142017:15)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async recordActivityInAccountPortal (/app/dist/index.js:141821:3)\n    at async Object.checkDayPass (/app/dist/index.js:141769:7)\n    at async /app/dist/index.js:144323:13\n    at async /app/node_modules/koa-compress/lib/index.js:38:5\n    at async errorHandling (/app/dist/index.js:136860:5)\n    at async userAgent (/app/node_modules/koa-useragent/dist/index.js:12:5)\n    at async session (/app/node_modules/koa-session/index.js:41:7)"},"pid":29,"tenantId":"default","correlationId":"838ccdd6-d00b-4f1e-a481-79b0328d8893","msg":""}
{"level":"ERROR","timestamp":"2024-12-10T09:51:51.588Z","service":"@budibase/worker","pid":29,"tenantId":"default","identityId":"us_f4ace1ef1ad74e538ae5a1fbc6116b0d","identityType":"user","correlationId":"793623da-1190-4a1a-a62d-8ed15bc84715","msg":"bb-alert: Error creating account user activity: Not Found"}
```

the last time we successfully logged in was probably last week.
and we've never made any changes in the evironments, budibase instances.

what can i do to fix this problem?

**Comment by mikesealey** [comment]
Hey @tooktok this could be an issue with our account portal. Please contact support@budibase.com with your tenant name and the email addresses of the affected accounts.
